packages/daemon/src/machines/SyncMachine.ts

```diff
@@ -137,6 +140,11 @@ export const SyncMachine = Machine<Context, any, Event>({
       states: {
         [CONNECTED_STATES.idle]: {
           id: CONNECTED_STATES.idle,
+          after: {
+            ACK_TIMEOUT: {
```

Comment:
```suggestion
            [ACK_TIMEOUT]: {
```

Shouldn't this be the act timeout value?

Reply:
It's a weird syntax, but no, this is a "referenced delay":

https://stately.ai/docs/delayed-transitions#referenced-delays

Reply:
Nice, naming it the same as the variable threw me off a little.